apertium_apy/mode_search.py

```diff
@@ -63,14 +65,20 @@ def search_path(rootpath, include_pairs=True, verbosity=1):
             for mtype, regex in type_re.items():
                 m = regex.match(filename)
                 if m:
-                    if mtype != 'pair':
-                        modename = m.group(1)  # e.g. en-es-anmorph
+                    if mtype == 'bilsearch':
+                        lang_src = to_alpha3_code(m.group(1))
+                        lang_trg = to_alpha3_code(m.group(2))
+                        lang_pair = f"{lang_src}-{lang_trg}"
+                        modename = f"{lang_pair}-bilsearch"
+                        dir_of_modes = os.path.dirname(dirpath)
+                        mode = (dir_of_modes, modename, lang_pair)
+                        modes[mtype].append(mode)
+                    elif mtype != 'pair':
+                        modename = m.group(1)
                         langlist = [to_alpha3_code(x) for x in m.group(2).split('-')]
-                        lang_pair = '-'.join(langlist)  # e.g. en-es
+                        lang_pair = '-'.join(langlist)
```

Comment:
@vykliuk, could you restore these comments (including the one up a few lines where modename is set)?